src/shell/components/FieldTypeUUID/FieldTypeUUID.js

```diff
@@ -20,37 +23,69 @@ export const FieldTypeUUID = React.memo(function FieldTypeUUID(props) {
 
   return (
     <label className={cx(styles.FieldTypeUUID, props.className)}>
-      <div className={styles.DateFieldTypeInput}>
-        <FontAwesomeIcon
-          className={styles.Icon}
-          icon={faClipboard}
-          aria-hidden="true"
-          title="Click to Copy"
-          onClick={(e) => {
-            const input = document.createElement("input");
-            document.body.appendChild(input);
-            input.value = props.value;
-            input.focus();
-            input.select();
-            const result = document.execCommand("copy");
-            input.remove();
-            if (result === "unsuccessful") {
-              return props.dispatch(
-                notify({
-                  type: "error",
-                  message: "Failed to copy the team ID to your clipboard",
-                })
-              );
-            }
+      <div>
+        <Tooltip
+          slotProps={{
+            popper: {
+              /**
+               * This is a custom style that is being applied to the tooltip
+               to adjust it's position not supported by Tooltip default props
+               */
+              sx: {
+                [`&.${tooltipClasses.popper}[data-popper-placement*="top"] .${tooltipClasses.tooltip}`]:
+                  {
+                    position: "relative",
+                    top: "47px",
+                  },
+              },
+            },
```

Comment:
@agalin920 

This solution I came up to position the tooltip inside the textfield. However on small screen this can sometime overlap with the text inside it. 

@zcolah  is that okay? or perhaps we can position it different way? 

@agalin920  maybe you have suggestion how to solve this properly?

Reply:
Updated on recent commit thank you! @agalin920